src/main/java/uk/gov/hmcts/cmc/claimstore/services/legaladvisor/DocAssemblyApi.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.cmc.claimstore.services.legaladvisor;
```

Comment:
should we consider extracting this api and client in separate  repository?

Reply:
We could! Just wondering if it's better to do at a later stage when things are a bit more stable